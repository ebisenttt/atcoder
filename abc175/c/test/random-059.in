607643316336036 711274536172978 525321020403320
