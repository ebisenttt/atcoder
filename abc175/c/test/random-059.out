442998724470604
